src/cdomains/apron/sharedFunctions.apron.ml

```diff
@@ -285,11 +289,38 @@ struct
     !expr
 
 
+  let lcm_den linexpr1 =
+    let exception UnsupportedScalar 
+    in
+    let frac_of_scalar scalar =
+      if Scalar.is_infty scalar <> 0 then (* infinity means unbounded *)
+        None
+      else match scalar with
+        | Float f -> if Stdlib.Float.is_integer f then Some (Q.of_float f) else None
+        | Mpqf f -> Some (Z_mlgmpidl.q_of_mpqf f)
+        | _ -> raise UnsupportedScalar
+    in
+    let extract_den (c:Coeff.union_5) _ =
```

Comment:
What's the point of this wildcard argument?

Reply:
That is a relic. Thanks for spotting.